### PR TITLE
Adding drop-downs and explanations to the API documentaion

### PR DESCRIPTION
**Pull request by LindsayYoung**
People from API testing asked for drop-downs. I am also adding some expanded explanations, those need to be formatted.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
It is a little off for `committee_type` in `/reports/{committee_type}` but overall it is an improvement.

File: webservices/args.py [hunk 1]
```diff
@@ -64,37 +64,50 @@ def one_of(value, options):
...
 committee = {
-    'year': Arg(int, multiple=True, description='A year that the committee was active- (after original registration date but before expiration date.)'),
-    'cycle': Arg(int, multiple=True, description='An election cycle that the committee was active- (after original registration date but before expiration date.)'),
-    'designation': Arg(str, multiple=True, description='The one-letter designation code of the organization'),
-    'organization_type': Arg(str, multiple=True, description='The one-letter code for the kind for organization'),
-    'committee_type': Arg(str, multiple=True, description='The one-letter type code of the organization'),
+    'year': Arg(int, multiple=True, description='A year that the committee was active- (After original registration date but before expiration date.)'),
+    'cycle': Arg(int, multiple=True, description='A 2-year election cycle that the committee was active- (after original registration date but before expiration date.)'),
+    'designation': Arg(str, multiple=True, enum=['', 'A', 'J', 'P', 'U', 'B', 'D'],
+        description='The one-letter designation code of the organization: \
```
Comment (jmcarp) [inline webservices/args.py]: If markdown works here, these might look better as lists:
* A: Authorized by a committee
* J: Joint fundraising committee
etc.

File: webservices/args.py [hunk 2]
```diff
@@ -64,37 +64,50 @@ def one_of(value, options):
 
 candidate_detail = {
     'cycle': Arg(int, multiple=True, description='Filter records to only those that were applicable to a given election cycle'),
-    'office': Arg(str, multiple=True, description='Governmental office candidate runs for'),
-    'state': Arg(str, multiple=True, description='U.S. State candidate is registered in'),
+    'office': Arg(str, multiple=True, enum=['', 'H', 'S', 'P'], description='Governmental office candidate runs for: House, Senate or President.'),
```
Comment (jmcarp) [inline webservices/args.py]: Let's pick one style for enumerating options (markdown lists, comma-separated lists, etc.) and use it consistently for fields like this.

File: webservices/args.py (hunk 1 above)
Comment (LindsayYoung) [inline webservices/args.py]: It wasn't rendering markdown, but now we have our own fork and I think I can just add the markdown class.

File: webservices/args.py (hunk 2 above)
Comment (LindsayYoung) [inline webservices/args.py]: :+1:

**Comment by jmcarp** [comment]
This is looking good! I made a few nitpick-y suggestions inline and pushed a one-character typo fix that was breaking npm.

**Comment by LindsayYoung** [comment]
I like your suggestions. 

Can the formatting be done in a separate request? I am going see if I can add markdown to those table cells. If I can, I will update the fork, and then reformat.

**Pull request merged**